python/servo/package_commands.py

```diff
@@ -234,15 +255,19 @@ def package(self, build_type: BuildType, android=None, target=None, flavor=None,
                 print("Deleting existing dmg")
                 os.remove(dmg_path)
 
+            # `hdiutil` gives "Resource busy" failures on GitHub Actions at times. This
+            # is an attempt to get around those issues by retrying the command a few times
+            # after a random wait.
             try:
-                subprocess.check_call(['hdiutil', 'create',
-                                       '-volname', 'Servo',
-                                       '-megabytes', '900',
-                                       dmg_path,
-                                       '-srcfolder', dir_to_dmg])
+                check_call_with_randomized_backoff(
+                    ['hdiutil', 'create', '-volname', 'Servo',
+                     '-megabytes', '900', dmg_path,
+                     '-srcfolder', dir_to_dmg],
+                    etries=3)
```

Comment:
```suggestion
                    retries=3)
```

Reply:
Thanks. I'm really surprised this didn't cause an error!